tests/core/test_core.py

```diff
@@ -124,18 +125,17 @@ def test_narrow_to_stream(
         mocker: MockerFixture,
         controller: Controller,
         index_stream: Index,
+        general_stream: Subscription,
         stream_id: int = 205,
         stream_name: str = "PTEST",
     ) -> None:
         controller.model.narrow = []
         controller.model.index = index_stream
         controller.view.message_view = mocker.patch("urwid.ListBox")
         controller.model.stream_dict = {
-            stream_id: {
-                "color": "#ffffff",
-                "name": stream_name,
-            }
+            stream_id: general_stream,
         }
+        controller.model.stream_dict[stream_id]["name"] = stream_name
```

Comment:
If it's necessary to keep this line for the test, can the stream_dict be removed instead later, when we switch to the new function?